app/controllers/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+  
+  def new
+  end
+end
```

Comment:
@shaniajubelag
Let's not include this yet, since this is not related to this task.

Reply:
The controller is not being used, so please remove file itself.